ct/x509/x509.go

```diff
@@ -66,6 +66,9 @@ func marshalPublicKey(pub interface{}) (publicKeyBytes []byte, publicKeyAlgorith
 			N: pub.N,
 			E: pub.E,
 		})
+		if err != nil {
+			return
```

Comment:
Should we return the err itself since we have a handle to it?

Reply:
`err` is defined in the function signature, so `return` is enough for `err` to be returned.

https://go.dev/play/p/VnE-UfpEdDE